<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Midivise</class>
 <widget class="QMainWindow" name="Midivise">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>400</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>500</width>
    <height>400</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>500</width>
    <height>400</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MIDivise</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="central_layout">
    <item>
     <widget class="QWidget" name="widget" native="true">
      <widget class="QCheckBox" name="checkBox_drums">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>180</y>
         <width>91</width>
         <height>22</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <weight>50</weight>
         <bold>false</bold>
        </font>
       </property>
       <property name="text">
        <string>Drums</string>
       </property>
      </widget>
      <widget class="QProgressBar" name="progressBar">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>343</y>
         <width>471</width>
         <height>20</height>
        </rect>
       </property>
       <property name="value">
        <number>24</number>
       </property>
      </widget>
      <widget class="QLabel" name="label_fileSelect">
       <property name="geometry">
        <rect>
         <x>100</x>
         <y>80</y>
         <width>371</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>No file selected (.wav/.mp3)</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_import">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>80</y>
         <width>75</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>Import File...</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_transcribe">
       <property name="geometry">
        <rect>
         <x>200</x>
         <y>310</y>
         <width>75</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>Transcribe...</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_2">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>110</y>
         <width>321</width>
         <height>31</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>10</pointsize>
         <weight>75</weight>
         <bold>true</bold>
         <underline>false</underline>
        </font>
       </property>
       <property name="text">
        <string>Transcription Settings:</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
       </property>
      </widget>
      <widget class="QComboBox" name="comboBox_drums">
       <property name="geometry">
        <rect>
         <x>160</x>
         <y>180</y>
         <width>151</width>
         <height>22</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>8</pointsize>
        </font>
       </property>
       <property name="whatsThis">
        <string/>
       </property>
       <property name="accessibleName">
        <string/>
       </property>
       <property name="accessibleDescription">
        <string/>
       </property>
       <property name="editable">
        <bool>false</bool>
       </property>
       <property name="currentText">
        <string>OaF Drums (recommended)</string>
       </property>
       <property name="maxVisibleItems">
        <number>8</number>
       </property>
       <item>
        <property name="text">
         <string>OaF Drums (recommended)</string>
        </property>
       </item>
      </widget>
      <widget class="QComboBox" name="comboBox_bass">
       <property name="geometry">
        <rect>
         <x>160</x>
         <y>220</y>
         <width>151</width>
         <height>22</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>8</pointsize>
        </font>
       </property>
       <item>
        <property name="text">
         <string>Basic Pitch (recommended)</string>
        </property>
       </item>
      </widget>
      <widget class="QCheckBox" name="checkBox_bass">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>220</y>
         <width>91</width>
         <height>22</height>
        </rect>
       </property>
       <property name="text">
        <string>Bass</string>
       </property>
      </widget>
      <widget class="QComboBox" name="comboBox_piano">
       <property name="geometry">
        <rect>
         <x>160</x>
         <y>260</y>
         <width>151</width>
         <height>22</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>8</pointsize>
        </font>
       </property>
       <item>
        <property name="text">
         <string>Onsets &amp; Frames (recommended)</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Basic Pitch</string>
        </property>
       </item>
      </widget>
      <widget class="QCheckBox" name="checkBox_piano">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>260</y>
         <width>91</width>
         <height>22</height>
        </rect>
       </property>
       <property name="text">
        <string>Piano/Others</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_3">
       <property name="geometry">
        <rect>
         <x>160</x>
         <y>140</y>
         <width>151</width>
         <height>31</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>10</pointsize>
         <weight>50</weight>
         <bold>false</bold>
         <underline>true</underline>
        </font>
       </property>
       <property name="text">
        <string>Transcription Model</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
       </property>
      </widget>
      <widget class="QLabel" name="label_4">
       <property name="geometry">
        <rect>
         <x>330</x>
         <y>140</y>
         <width>141</width>
         <height>31</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>10</pointsize>
         <weight>50</weight>
         <bold>false</bold>
         <underline>true</underline>
        </font>
       </property>
       <property name="text">
        <string>MIDI Program Change</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
       </property>
      </widget>
      <widget class="QLabel" name="label_5">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>140</y>
         <width>131</width>
         <height>31</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>10</pointsize>
         <weight>50</weight>
         <bold>false</bold>
         <underline>true</underline>
        </font>
       </property>
       <property name="text">
        <string>Instrument Selection</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
       </property>
      </widget>
      <widget class="Line" name="line">
       <property name="geometry">
        <rect>
         <x>140</x>
         <y>150</y>
         <width>20</width>
         <height>151</height>
        </rect>
       </property>
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
      </widget>
      <widget class="Line" name="line_2">
       <property name="geometry">
        <rect>
         <x>310</x>
         <y>150</y>
         <width>20</width>
         <height>151</height>
        </rect>
       </property>
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
      </widget>
      <widget class="QSpinBox" name="spinBox_bass">
       <property name="geometry">
        <rect>
         <x>330</x>
         <y>220</y>
         <width>141</width>
         <height>22</height>
        </rect>
       </property>
       <property name="maximum">
        <number>127</number>
       </property>
       <property name="value">
        <number>32</number>
       </property>
      </widget>
      <widget class="QSpinBox" name="spinBox_piano">
       <property name="geometry">
        <rect>
         <x>330</x>
         <y>260</y>
         <width>141</width>
         <height>22</height>
        </rect>
       </property>
       <property name="maximum">
        <number>127</number>
       </property>
      </widget>
      <widget class="QLabel" name="label_6">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="geometry">
        <rect>
         <x>330</x>
         <y>170</y>
         <width>141</width>
         <height>41</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>8</pointsize>
         <italic>true</italic>
        </font>
       </property>
       <property name="text">
        <string>Uses MIDI Channel 10 Standard (Percussion)</string>
       </property>
       <property name="wordWrap">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QLabel" name="label_7">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>481</width>
         <height>51</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Yu Mincho</family>
         <pointsize>28</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>MIDivise</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
      <widget class="QLabel" name="progressTooltip">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>360</y>
         <width>441</width>
         <height>23</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <italic>true</italic>
        </font>
       </property>
       <property name="text">
        <string>Please configure transcription settings to proceed with audio processing...</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_9">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>50</y>
         <width>321</width>
         <height>31</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>10</pointsize>
         <weight>75</weight>
         <bold>true</bold>
         <underline>false</underline>
        </font>
       </property>
       <property name="text">
        <string>Select Source Audio Track:</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
       </property>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
